modules/twitch-notifications/configs/config.json

```diff
@@ -0,0 +1,58 @@
+{
+    "filename": "config.json",
+    "content": [
+      {
+        "field_name": "liveMessage",
+        "default": "Hey, %streamer% is live on Twitch streaming %game%! Check it out at %url% !",
```

Comment:
```suggestion
        "default": "Hey, %streamer% is live on Twitch streaming %game%! Check it out at %url%!",
```
remove space

Reply:
This space is required because otherwise the exclamation mark belongs to the url